geode-docs/topologies_and_comm/multi_site_configuration/setting_up_a_multisite_system.html.md.erb

```diff
@@ -160,7 +160,18 @@ The gateway sender configuration for a specific sender `id` must be identical on
 
 ## <a id="setting_up_a_multisite_system__section_E1DEDD0743D54831AFFBCCDC750F8879" class="no-quick-link"></a>Create Data Regions for Multi-site Communication
 
-When you are using a multi-site configuration, you choose which data regions to share between sites. Because of the high cost of distributing data between disparate geographical locations, not all changes are passed between sites. After you define gateway senders, configure regions to use the gateway sender to distribute region events.
+When you are using a multi-site configuration, you choose which data regions to share between sites. Because of the high cost of distributing data between disparate geographical locations, not all changes are passed between sites. 
```

Comment:
Delete "you are" from "When you are using a multi-site configuration ..." (the "you" in "you choose" covers it).